patchwork-events-entity/src/main/java/net/minecraftforge/event/entity/living/EntityMountEvent.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2020, 2019-2020
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.event.entity.living;
+
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.entity.EntityEvent;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.world.World;
+
+/**
+* This event gets fired whenever a entity mounts/dismounts another entity.<br>
+* <b>entityBeingMounted can be null</b>, be sure to check for that.
+* <br>
```

Comment:
Please switch line breaks to proper paragraphs

Reply:
wdym by proper paragraphs?

Reply:
nevermind, I just looked at a different event

Reply:
When I add the correct spacing, checkstyle fails for a "non-tab indent"
Even though it's literally a single space